content/pt-br/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,392 @@
+
+
+{{< note >}}
+Como mencionado na seção de [singularidade do nome do nó](#singularidade-de-nome-do-no), quando a configuração do nó precisa ser atualizada, é uma boa prática registrar novamente o nó no servidor da API. Por exemplo, se o kubelet estiver sendo reiniciado com o novo conjunto de `--node-labels`, mas o mesmo nome de nó for usado, a alteração não entrará em vigor, pois os labels estão sendo definidos no registro do Nó.
+
+Pods já agendados no Nó podem se comportar mal ou causar problemas se a configuração do Nó for alterada na reinicialização do kubelet. Por exemplo, o Pod já em execução pode estar contaminado contra os novos rótulos atribuídos ao Nó, enquanto outros Pods, que são incompatíveis com esse Pod, serão agendados com base nesse novo rótulo. O novo registro do nó garante que todos os Pods sejam drenados e devidamente reprogramados.
+{{< /note >}}
+
+### Administração manual de nós
+
+Você pode criar e modificar objetos Nó usando o {{< glossary_tooltip text="kubectl" term_id="kubectl" >}}..
+
+Quando você quiser criar objetos Nó manualmente, defina a opção do kubelet `--register-node=false`.
+
+Você pode modificar objetos Nó, independentemente da configuração de `--register-node`. Por exemplo, você pode definir labels em um nó existente ou marcá-lo como não programado.
+
+Você pode usar labels nos Nós em conjunto com seletores de nós nos Pods para controlar o agendamento. Por exemplo, você pode restringir um Pod a ser elegível apenas para ser executado em um subconjunto dos nós disponíveis.
+
+Marcar um nó como não programável impede que o agendador coloque novos pods nesse nó, mas não afeta os Pods existentes no nó. Isso é útil como uma etapa preparatória antes da reinicialização de um nó ou outra manutenção.
+
+Para marcar um nó como não programado, execute:
+
+```shell
+kubectl cordon $NODENAME
+```
+
+Consulte [Drenar um nó com segurança](/docs/tasks/administer-cluster/safely-drain-node/) para obter mais detalhes.
+
+{{< note >}}
+Os Pods que fazem parte de um {{< glossary_tooltip term_id="daemonset" >}} toleram ser executados em um nó não programável. Os DaemonSets geralmente fornecem serviços locais de nós que devem ser executados em um Nó, mesmo que ele esteja sendo drenado de aplicativos de carga de trabalho.
+{{< /note >}}
+
+## Situação do Nó
+
+O status de um nó contém as seguintes informações:
+
+* [Endereços](#addresses)
+* [Condições](#condition)
+* [Capacidade](#capacity)
+* [Informação](#info)
+
+Você pode usar o `kubectl` para visualizar o status de um nó e outros detalhes:
+
+```shell
+kubectl describe node <insert-node-name-here>
+```
+
+Cada seção da saída está descrita abaixo.
+
+### Endereços
+
+O uso desses campos pode mudar dependendo do seu provedor de nuvem ou configuração `bare metal`.
```

Comment:
How about using "configuração dedicada" instead of using the English term bare metal in this context?